birdhouse/components/cowbird/default.env

```diff
@@ -0,0 +1,29 @@
+# -------
+# Cowbird
+# -------
+
+# add any new variables not already in 'VARS' or 'OPTIONAL_VARS' that must be replaced in templates here
+# single quotes are important in below list to keep variable names intact until 'pavics-compose' parses them
+EXTRA_VARS='
```

Comment:
This extra housekeeping blocks about VARS and EXTRA_VARS could be at the bottom of this file so we immediately see the most important values first.

Reply:
I've added it at the top to actually produce the inverse effect. Something like a reminder because you see it first as soon as you open the file. When at the bottom, it is very easy to add new variables to the file and forget to add them to this list for them to become effective.

Reply:
Ah I see, makes sense.